code/modules/client/preferences_savefile.dm

```diff
@@ -49,55 +49,12 @@
 				break
 		return FALSE
 
-	if(savefile_version < 22)
-		WRITE_FILE(S["windowflashing"], TRUE)
-
-	if(savefile_version < 24)
-		WRITE_FILE(S["menuoptions"], list())
-
-	if(savefile_version < 25)
-		WRITE_FILE(S["ghost_vision"], TRUE)
-		WRITE_FILE(S["ghost_orbit"], GHOST_ORBIT_CIRCLE)
-		WRITE_FILE(S["ghost_form"], GHOST_DEFAULT_FORM)
-		WRITE_FILE(S["ghost_others"], GHOST_OTHERS_DEFAULT_OPTION)
-
-	if(savefile_version < 27)
-		switch(S["ui_style"])
-			if("Orange")
-				WRITE_FILE(S["ui_style"], "Plasmafire")
-			if("old")
-				WRITE_FILE(S["ui_style"], "Retro")
-		if(S["ui_style_alpha"] > 230)
-			WRITE_FILE(S["ui_style_alpha"], 230)
-
-	if(savefile_version < 28)
-		WRITE_FILE(S["tooltips"], TRUE)
-
-	if(savefile_version < 29)
-		WRITE_FILE(S["metadata"], null)
-		WRITE_FILE(S["jobs_low"], null)
-		WRITE_FILE(S["jobs_medium"], null)
-		WRITE_FILE(S["jobs_high"], null)
-		WRITE_FILE(S["job_preferences"], list())
-
-	if(savefile_version < 32)
-		WRITE_FILE(S["observer_actions"], TRUE)
-
-	if(savefile_version < 35)
-		WRITE_FILE(S["focus_chat"], FALSE)
-
-	if(savefile_version < 36)
-		WRITE_FILE(S["clientfps"], 0)
-
-	if(savefile_version < 37)
-		WRITE_FILE(S["ai_name"], "ARES v3.2")
-
-	if(savefile_version < 38)
-		WRITE_FILE(S["menuoptions"], list())
-
 	if(savefile_version < 39)
 		WRITE_FILE(S["toggles_gameplay"], toggles_gameplay)
 
+	if(savefile_version < 41)
+		WRITE_FILE(S["toggles_gameplay"], toggles_gameplay)
+
```

Comment:
These two actions are repeated. Why do we have them anyway, why are we increasing the savefile size and why removing the other ones?

Reply:
This was a bad copy-paste from me, it should be the new vars for runechat

I also removed the older settings since they have been there for over 9months and generally only for players that may be returning after that time and we want to progressively improve their prefs, however since then much has changed and they would be better off just having a fresh pref file.